FaceTracking/ViewController.swift

```diff
@@ -228,12 +239,37 @@ extension ViewController: AVCaptureVideoDataOutputSampleBufferDelegate {
 
 extension ViewController {
     func update(with faceRect: CGRect, text: String) {
+        
         DispatchQueue.main.async {
-            UIView.animate(withDuration: 0.2) {
+            UIView.animate(withDuration: 0.2) { [unowned self] in
                 self.detailsView.detailsLabel.text = text
                 self.detailsView.alpha = 1.0
                 self.detailsView.frame = faceRect
+                
+                self.rotateDetailViewWithDeviceOrientation()
             }
         }
     }
+    
+    func rotateDetailViewWithDeviceOrientation() {
+        switch UIDevice.current.orientation {
```

Comment:
change formatting and remove new lines

Reply:
Hope I did this correctly.

Reply:
Should I remove the call to rotate the detailView with the device orientation?